site/_resources/upgrading.md

```diff
@@ -8,23 +8,82 @@ layout: page
-<li>The contour_httpproxy_valid_total Gauge metric has been removed in favor of the similarly named contour_httpproxy_valid Gauge metric.</li>
-<li>The contour_httpproxy_orphaned_total Gauge metric has been removed in favor of the similarly named contour_httpproxy_orphaned Gauge metric.</li>
-</ul>
+<b>Deprecation Notices</b>
+<br>
 </div>
 
 <br>
 
 <div id="toc" class="navigation"></div>
 
-## Upgrading Contour 1.14.0 to 1.14.1
+## Upgrading Contour 1.14.1 to 1.15.0
 
-Contour 1.14.1 is the current stable release.
+Contour 1.15.0 is the current stable release.
+
+### Required Envoy version
+
+All users should ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.18.2`.
+
+Please see the [Envoy Release Notes][31] for information about issues fixed in Envoy 1.18.2.
+
+### The easy way to upgrade
+
+If the following are true for you:
+
+* Your installation is in the `projectcontour` namespace.
+* You are using our [quickstart example][18] deployments.
+* Your cluster can take a few minutes of downtime.
+
+Then the simplest way to upgrade to 1.15.0 is to delete the `projectcontour` namespace and reapply one of the example configurations:
+
+```bash
+$ kubectl delete namespace projectcontour
+$ kubectl apply -f {{site.url}}/quickstart/v1.15.0/contour.yaml
+```
+
+This will remove the Envoy and Contour pods from your cluster and recreate them with the updated configuration.
+If you're using a `LoadBalancer` Service, (which most of the examples do) deleting and recreating may change the public IP assigned by your cloud provider.
+You'll need to re-check where your DNS names are pointing as well, using [Get your hostname or IP address][12].
+
+### The less easy way
+
+This section contains information for administrators who wish to apply the Contour 1.14.1 to 1.15.0 changes manually.
+The YAML files referenced in this section can be found by cloning the Contour repository and checking out the `v1.15.0` tag.
+
+If your version of Contour is older than v1.14, please upgrade to v1.14 first, then upgrade to v1.15.0.
+
+1. The Contour CRD definitions must be re-applied to the cluster, since a number of compatible changes and additions have been made to the Contour API:
+
+    ```bash
+    $ kubectl apply -f examples/contour/01-crds.yaml
+    ```
+
+1. Users of the example deployment should reapply the certgen Job YAML which will re-generate the relevant Secrets in the new format, which is compatible with [cert-manager](https://cert-manager.io) TLS secrets.
+This will rotate the TLS certificates used for gRPC security.
+
+    ```bash
+    $ kubectl apply -f examples/contour/02-job-certgen.yaml
+    ```
+
+1. This release includes an update to RBAC rules. Update the Contour ClusterRole with the following:
+
+    ```bash
+    $ kubectl apply -f examples/contour/02-role-contour.yaml
+    ```
+
+1. This release includes changes to support Ingress wildcard hosts that require Envoy to be upgraded *before* Contour. Update the Envoy DaemonSet:
```

Comment:
I put it this way to not overcomplicate it since it doesn't seem to be an issue to always update Envoy first, but I can add that caveat if you prefer, LMK.

Reply:
nah, seems ok to leave as is tbh, we've already had that before i think so we're not breaking anything, keeps it simpler